### PR TITLE
Pre Quote Component - UI

### PR DESCRIPTION
**Pull request by nalancer08**
### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### Linked issue

- [X] Not tracked

### Why
Pre Quote Component in TradeFlow -- Only UI interaction with Asset Service and Price Service

### Evidence

imagen

imagen

imagen

imagen

imagen

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -8,4 +8,6 @@
 
     <string name="activity_list_prices_assets_component_title">Componente: Lista Crypto</string>
     <string name="activity_list_prices_assets_component_sub_title">Componente: Lista Crypto - Estilos editables</string>
+
+    <string name="activity_trade_flow_title">Comprando Bitcoin</string>
```
[the remark this answers is not in the document]
Comment (nalancer08) [reply app/src/main/res/values-es/strings.xml]: Solved!! This line of String was not used!!

File: version.gradle [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ext {
-    sdk_version = '0.0.0'
+    sdk_version = '0.0.6'
```
Comment (brent-cybrid) [inline version.gradle:2]: Why's this here?

Comment (nalancer08) [reply version.gradle:2]: To release a new version manually, cause we have to fix the script for get the new tag

File: SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/ListPricesView.kt [hunk 3]
```diff
@@ -333,7 +336,7 @@ fun CryptoAssetItem(crypto: SymbolPriceBankModel,
                     index:Int, selectedIndex:Int,
                     context: Context? = null,
                     customStyles: ListPricesViewCustomStyles,
-                    onClick: (Int) -> Unit) {
+                    onClick: (AssetBankModel, AssetBankModel) -> Unit) {
```
Comment (jc-cybrid) [inline SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/ListPricesView.kt:339]: Could you add labels to the closure properties? It's not self explanatory on the purpose of each property.

Comment (jc-cybrid) [inline SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/ListPricesView.kt:339]:
```suggestion
                    onClick: (asset: AssetBankModel, pairAsset: AssetBankModel) -> Unit) {
```

File: SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/listprices/view/ListPricesViewModel.kt [hunk 4]
```diff
@@ -86,4 +93,17 @@ class ListPricesViewModel : ViewModel() {
         val a = assets.find { it.code == symbol }
         return a
     }
+
+    fun getBuyPrice(symbol: String): SymbolPriceBankModel {
+
+        var ret = SymbolPriceBankModel()
```
Comment (jc-cybrid) [inline SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/listprices/view/ListPricesViewModel.kt:99]: `ret` stands for...?

**Review by brent-cybrid — APPROVED** [review]

**Pull request merged**